webpack.config.js

```diff
@@ -55,4 +57,13 @@ module.exports = {
         new webpack.optimize.UglifyJsPlugin(),
         new webpack.optimize.DedupePlugin(),
     ],
+    module: {
+        loaders: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components|javascripts-legacy|components|vendor)/,
```

Comment:
so, seems not... :( 

it's also not how babel demo the loader working: https://github.com/babel/babel-loader#usage